code/modules/surgery/organs/augments_internal.dm

```diff
@@ -128,15 +126,16 @@
 	implant_color = "#FFFF00"
 	slot = "brain_antistun"
 	origin_tech = "materials=5;programming=4;biotech=5"
+	var/stun_max_amount = 2
```

Comment:
Why is this made into a variable?

Reply:
Originally, the hardened implant was going to have a different value here. Even without that, I don't see why a value like that should be a define rather than a var, this way it's easier to make more versions.